tests/conftest.py

```diff
@@ -38,10 +38,10 @@ def feature_properties() -> dict:
     fp["P_ID"] = flags.IS_UNORDERED
     fp["PG_ID_3"] = flags.IS_UNORDERED
     fp["L_ID"] = flags.IS_UNORDERED
-    fp["dayofweek"] = flags.IS_ORDERED
+    fp["dayofweek"] = flags.IS_UNORDERED
```

Comment:
Ignorant question, why are "dayofweek" and "PROMOTION_TYPE" unordered? I would have thought that the former is ordered

Reply:
Yes, you're right, day of week is ordered. However, it is a cyclical ordering, i.e., day 0 should be smoothed by day 6. And this is not done (intentionally) with the implementation of ordered smoothing here. So, I think unordered is better.